### PR TITLE
(Litmus-Portal): Changed the litmus version to v2.0.0-RC1

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

**This PR will update the litmus version to v2.0.0-RC1** 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by ishangupta-ds — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

File: docs/2.0.0-Beta/litmus-2.0.0-Beta.yaml [hunk 1]
```diff
@@ -143,6 +143,9 @@ rules:
   - apiGroups: [litmuschaos.io]
     resources: [chaosengines, chaosexperiments, chaosresults]
     verbs: [get, create, update, patch, delete, list, watch, deletecollection]
+  - apiGroups: [apiextensions.k8s.io]
+    resources: [customresourcedefinitions]
+    verbs: [list, get]
```
Comment (uditgaurav) [inline docs/2.0.0-Beta/litmus-2.0.0-Beta.yaml:148]: can we remove it from ns where you have added?

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
@Jonsy13 can we update all images in monitoring directory as well? And all 1.13.3 or 1.13.6 to RC

**Review by ispeakc0de — APPROVED** [review]

**Review by gdsoumya — APPROVED** [review]

File: docs/2.0.0-Beta/litmus-2.0.0-Beta.yaml (hunk 1 above)
Comment (Jonsy13) [reply docs/2.0.0-Beta/litmus-2.0.0-Beta.yaml:148]: Yes, it was going to be added in this PR only. I have reverted my changes.. It is added only for cluster scope now.

**Review by ishangupta-ds — APPROVED** [review]
Images and version can be updated at other places as well before merging, looks good for chaos center.

**Pull request merged**